### PR TITLE
Update BeethovenLv-O31

### PR DESCRIPTION
**Pull request by AlexSchr**

### CODE REVIEW

**Comment by dominicus** [comment]
The following changes confirmed:
Bar71: treble 1st beat (f' a' c'')8->(f' a' f'')8
Bar86: treble 2nd beat g->ges

@AlexSchr, requesting an additional commit to improve on the following:
Bar49, treble 1st voice: add closing slur ")" on ees (2nd beat)
Bar88, treble: (ges a) chord at beat 2.5 is tied to the corresponding notes in beat 3.  Cross-voice ties can get hairy, so feel free to take a pass on this update.

**Comment by AlexSchr** [comment]
@dominicus: I added your suggested corrections. 
In bar 88, I added an a in the lower voice. Feel free to improve if you know a better solution.

File: ftp/BeethovenLv/O31/LVB_Sonate_31no2_2/LVB_Sonate_31no2_2.ly [hunk 1]
```diff
@@ -115,7 +115,7 @@
  << { fis'4^\p g'4. bes'8 } \\ { a4\rest <bes ees'>2 } >>
  << { ees''2( c''8 a') } \\ { f'4\rest a'!4\sf\>~ a'8\! s8 } \\ { \stemDown s4 <ees' ges'>2 } >>
  << { c''4\p( bes'4. d''8) } \\ { f'4\rest <d' f'>4_\markup { \italic "cresc." } <f' aes'> } >> \break
- << { f''8( ees''4.\sf <a' c''>8( a') } \\ { e'4\rest <g' bes'>8[ <ges' a'!>\sf <ees' ges'> <c' ees' ges'>] } >>
+ << { f''8( ees''4.\sf <a' c''>8 a') } \\ { e'4\rest <g' bes'>8[ <ges'~ a'!~>\sf <ees' ges' a'> <c' ees' ges'>] } >>
```
Comment (dominicus) [inline ftp/BeethovenLv/O31/LVB_Sonate_31no2_2/LVB_Sonate_31no2_2.ly]: Try:
`<< { f''8( ees''4.*2/3\sf \tweak Stem.transparent ##t a'8~ <a' c''>8[ a']) } \\ { e'4\rest <g' bes'>8[ <ges'~ a'!>\sf <ees' ges'> <c' ees' ges'>] } >>`

**Comment by dominicus** [comment]
Thumbs up, compiles clean on my environment.  Thank You!

**Pull request merged**